javascriptlib/src/mill/javascriptlib/testmods/TestModule.scala

```diff
@@ -0,0 +1,33 @@
+package mill.javascriptlib.testmods
```

Comment:
No need to put this inside `javascriptlib.testmods`, having it inside the top-level `javascriptlib` is probably fine